client/extensions/woocommerce/index.js

```diff
@@ -178,3 +183,6 @@ export default function() {
 		page( '/store/stats/:type/:unit/:site', siteSelection, navigation, StatsController );
 	}
 }
+
+initExtension();
```

Comment:
I'm not sure. Maybe? I'll add a TODO to this line to remind us to revisit this approach. I'm definitely up for something better here.